sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageBuilder.java

```diff
@@ -91,7 +91,11 @@ public AzureBlobStorageImpl build() {
         if (this.url != null) {
             client.setUrl(this.url);
         }
-        client.setVersion(this.version);
+        if (this.version != null) {
```

Comment:
We allow customer to customize the service version through builder.
https://github.com/Azure/azure-sdk-for-java/blob/e2a2ad88e6f6c9511ebd710a2622cb4626ac91c4/sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java#L133

If not specified, this "2019=02-02"

Reply:
This is done by codegen and functionally does the same thing our hand crafted builders do